app/Http/Controllers/V2/Dashboard/ViewRestorationStrategyController.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace App\Http\Controllers\V2\Dashboard;
+
+use App\Helpers\TerrafundDashboardQueryHelper;
+use App\Http\Controllers\Controller;
+use App\Http\Resources\V2\Dashboard\ViewRestorationStrategyResource;
+use App\Models\V2\Sites\Site;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+
+class ViewRestorationStrategyController extends Controller
+{
+    public function __invoke(Request $request): JsonResponse
+    {
+        $query = $this->buildProjectQuery($request);
+
+        $projectIds = $query->pluck('v2_projects.id')->toArray();
+
+        $restorationStrategy = $this->getRestorationStrategy($projectIds);
+
+        $landUseType = $this->getLandUseType($projectIds);
+
+        $landTenures = $this->getLandTenures($projectIds);
+
+        $result = [
+            'restorationStrategies' => $this->getResultArray($restorationStrategy, 'strategy'),
+            'landUseTypes' => $this->getResultArray($landUseType, 'land_use'),
+            'landTenures' => $this->getResultArray($landTenures, 'land_tenure'),
+        ];
+
+        return new JsonResponse(ViewRestorationStrategyResource::make($result));
+    }
+
+    private function buildProjectQuery(Request $request)
+    {
+
+        $query = TerrafundDashboardQueryHelper::buildQueryFromRequest($request);
+
+        return $query;
+    }
+
+    private function getRestorationStrategy(array $projectIds)
+    {
+        $strategies = ['direct-seeding', 'tree-planting', 'assisted-natural-regeneration'];
+
+        $conditions = implode(' OR ', array_map(function ($strategy) {
+            return "JSON_UNQUOTE(JSON_EXTRACT(restoration_strategy, CONCAT('\$[', numbers.n, ']'))) = '$strategy'";
+        }, $strategies));
+
+        $numbers = implode(' UNION ALL ', array_map(function ($n) {
+            return "SELECT $n AS n";
+        }, range(0, 3)));
+
+        return DB::table(DB::raw("(SELECT DISTINCT
+            project_id,
+            JSON_UNQUOTE(JSON_EXTRACT(restoration_strategy, CONCAT('\$[', numbers.n, ']'))) AS strategy
+        FROM
+            v2_sites
+        CROSS JOIN
+            ($numbers) numbers
+        WHERE
+            project_id IN (" . implode(',', $projectIds) . ")
+            AND ($conditions)
+        ) AS subquery"))
+            ->groupBy('strategy')
+            ->select('strategy', DB::raw('COUNT(*) as count_per_project'))
+            ->get();
+    }
+
+    private function getLandUseType(array $projectIds)
+    {
+        $landUseTypes = ['agroforest', 'open-natural-ecosystem', 'mangrove', 'natural-forest', 'peatland', 'riparian-area-or-wetland', 'silvopasture', 'urban-forest', 'woodlot-or-plantation'];
```

Comment:
Same here as above; please move to a class-level const.

Reply:
moved